jaeger-jaxrs2/src/test/java/com/uber/jaeger/filters/jaxrs2/ServerFilterTest.java

```diff
@@ -97,8 +99,8 @@ public void filter() throws Exception {
     Span span = spans.get(0);
     Map<String, Object> tags = span.getTags();
 
-    assertTrue(span.isRPC() && !span.isRPCClient());
-    assertEquals(source, span.getPeer().getService_name());
+    assertEquals(method, span.getOperationName());
+    assertEquals(Tags.SPAN_KIND_SERVER, tags.get(Tags.SPAN_KIND.getKey()));
```

Comment:
I would have to change `InMemoryReporter` to hold tracer tags. Better is to test in in `TracerTagsTest`